tex/chapters/03-Theory.tex

```diff
@@ -3,88 +3,152 @@
+Instead of the original behaviour, it makes move $C$ with probability $x$, move
+$D$ with probability $y$, and otherwise uses the response appropriate to
+strategy $A$ (assuming $x+y < 1$).
 \end{definition}
 
 The notation $\JA$ comes from the initials of the names Joss and Anne.
 Joss was a strategy submitted to one of Axelrod’s original tournaments and it would occasionally defect without provocation in the hopes of a slight improvement in score.
 Anne is the first name of A. Stanley who suggested the addition of random cooperation instead of random defection \cite{Ashlock2008}.
-When $x + y = 1$, the original strategy is not used, and the resulting behaviour is a random strategy with probabilities $(x, y)$.
-In more general terms, a $\JA$ strategy is an alteration of a strategy $A$ that causes the strategy to be played with random noise inserted into the responses.
+When $x + y = 1$, the original strategy is not used, and the resulting behaviour
+is a random strategy with probabilities $(x, y)$. Note that this corresponds to
+Random($x$) as described in Section~\ref{ssec:strat_random}.
+In more general terms, a $\JA$ strategy is an alteration of $A$ that causes the strategy to be played with random noise inserted into the responses.
 
-\section{The Dual}\label{sec:dual}
-The Dual is another, more complex transformation of a strategy (note that transformations can be applied over each other).
-Given a history for an opponent, the responses of the original strategy and the dual would be opposite.
 
-\begin{definition}\label{def:dual}
-Strategy $A'$ is said to be the \textbf{Dual} of strategy $A$ if $A$ and $A'$ can be written as finite-state machines that are identical except that their responses are reversed.
-\end{definition}
 
-It is important to note that this is different to taking a strategy and flipping it's responses.
-The dual relies on knowledge of the underlying state of the original strategy, whereas the flip does not.
-This is shown in Table \ref{tab:strat-dual-flip}.
-For an outline of how Pavlov plays, see Section \ref{ssec:strat_pavlov}.
 
 \section{The Flip}\label{sec:flip}
 The Flip is a simple transformation that returns the opposite of the strategy.
-This is subtly different from the Dual as demonstrated below.
+This is subtly different from the Dual as demonstrated in Table \ref{tab:strat-dual-flip}.
 
 \begin{definition}\label{def:flip}
 Strategy $A'$ is said to be the \textbf{Flip} of strategy $A$ if, $A$ and $A'$ return different actions for the same game history (includes opponent and self).
 \end{definition}
 
+
+
+
+\section{The Dual}\label{sec:dual}
+The Dual is another, more complex transformation of a strategy (note that both
+these transformations can be applied commutatively).
+
+\begin{definition}\label{def:dual}
+Strategy $A'$ is said to be the \textbf{Dual} of strategy $A$ if $A$ and $A'$ can be written as finite-state machines that are identical except that their responses are reversed.
```

Comment:
This can be improved:

- State that traditionally (cite) the Dual is only defined on a strategy with an FSM representation.
- Then give the definition, say say that the Dual $A'$ of a strategy $A$ with an FSM representation has the same FSM representation except that all responses are reversed.

You can (after your current paragraph) then say that in Section X you will be extending this definition to allow for the Dual to be calculated for **any** strategy (even ones without a natural FSM representation).